sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileGetRangeListDiffOptions.cs

```diff
@@ -35,7 +35,10 @@ public class ShareFileGetRangeListDiffOptions
         /// changed between target file and previous snapshot.  Changed ranges
         /// include both updated and cleared ranges. The target file may be a
         /// snapshot, as long as the snapshot specified by
-        /// previousSnapshot is the older of the two.
+        /// <see cref="PreviousSnapshot"/> is the older of the two.
+        /// For more information on working with share snapshots,
+        /// <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/create-share">
```

Comment:
This link is how to Create a Share in general right? I couldn't find anything snapshot specific except for  virtual directory access which isn't the same as previous snapshot

Reply:
Did you mean
https://learn.microsoft.com/en-us/azure/storage/files/storage-snapshots-files

or 

https://learn.microsoft.com/en-us/rest/api/storageservices/list-ranges

Reply:
@amnguye good question. I used that link since it was the link that was used for the Snapshot property
https://github.com/Azure/azure-sdk-for-net/pull/45870/files#diff-f923c18ac238f02fbb0d40ef587b5c2e54980865485809e90527ea25efb49b3aL22

It does seem like there is not much info about snapshots in that link. I do like the first link you provided so I will use that

Reply:
Yeah seems like a mistake above.